common/utils.go

```diff
@@ -75,12 +75,11 @@ func SetupLOG(logger *l.Logger, logLevel string) {
 	switch os.Getenv("INITSYSTEM") {
 	case "SYSTEMD":
 		fileFormat = l.MustStringFormatter(
-			`%{level:.4s} %{id:03x} %{shortfunc} > %{message}`)
+			`%{level:.4s} %{message}`)
```

Comment:
There's the `callpath` to show you the callpath of that log message, and the `shortpkg` to print the package of that functioname.